CodeStarterCamp_Week4/step01.swift

```diff
@@ -10,20 +10,53 @@ import Foundation
 struct Activity {
     let name: String
     let action: (BodyCondition) -> Void
+    
 }
 
 class BodyCondition {
-    var upperBodyStrength: Int //상체근력
-    var lowerBodyMuscleStrength: Int //하체근력
-    var muscularEndurance: Int //근지구력
-    var degreeOfFatigue: Int //피로도
+    var upperBodyStrength: Int {
+        didSet {
+            print("상체근력이 \(printChangeValue(self.upperBodyStrength - oldValue))")
+        }
+    }
+    var lowerBodyMuscleStrength: Int{
+        didSet {
+            print("하체근력이 \(printChangeValue(self.lowerBodyMuscleStrength - oldValue))")
+        }
+    }
+    var muscularEndurance: Int{
+        didSet {
+            print("근지구력이 \(printChangeValue(self.muscularEndurance - oldValue))")
+        }
+    }
+    var degreeOfFatigue: Int{
+        didSet {
+            print("피로도가 \(printChangeValue(self.degreeOfFatigue - oldValue))")
+        }
+    }
+    
+    func printChangeValue(_ changeValue: Int) -> String {
+        let changeValueText: String
+        
+        if changeValue > 0 {
+            changeValueText = "\(changeValue) 상승합니다."
+        } else if changeValue < 0 {
+            changeValueText = "\(changeValue) 하락합니다."
+        } else {
+            changeValueText = "변화 없습니다."
+        }
+        
+        return "\(changeValueText)"
```

Comment:
고냥 변수를 리턴해주셔도 됩니다 ㅎㅎ
```suggestion
        return changeValueText
```

Reply:
요 부분 말고도 어제 PR보내고 나서 코드 점검 하는데 밝혀진 오류도 있고 또 너무 쓸데없이 print문이랑  try도 남발한 부분이 있어서 함께 수정했습니다.

무슨 생각으로 그렇게 했는지.. 지금 보니.. 코드가 난리가 아니네요..ㅠ_ㅠ